py/server/deephaven/__init__.py

```diff
@@ -21,6 +21,6 @@
 from .csv import write as write_csv
 from .stream.kafka import consumer as kafka_consumer
 from .stream.kafka import producer as kafka_producer
-from .table_factory import empty_table, time_table, merge, merge_sorted, new_table, DynamicTableWriter, InputTable
+from .table_factory import empty_table, time_table, merge, merge_sorted, new_table, DynamicTableWriter, input_table
```

Comment:
We should clearly mark this as a breaking change, and include in the release notes. Is there a process for that in core (@rcaudy)?

For web-client-ui, we label the PR `breaking` and include in the PR description details about the breakage/migration instructions: https://github.com/deephaven/web-client-ui/pull/881
These then get included in the release notes: https://github.com/deephaven/web-client-ui/releases/tag/v0.22.0

Reply:
Sounds like a fine process. Looks like @jmao-denver has tagged this "ReleaseNotesNeeded". I added "breaking" just now.